crate/content.md

```diff
@@ -1,90 +1,113 @@
 ```
 
-Any configuration settings may be specified upon startup using the `-D` option prefix. For example, configuring the cluster name by using system properties will work this way:
+For production use it's strongly recommended to use only one container per machine. This will give the best possible performance and by mapping the ports from the Docker container to the host it acts like a native installation. Crate's default ports 4200 (HTTP) and 4300 (Transport protocol).
 
 ```console
-$ docker run -d -p 4200:4200 -p 4300:4300 crate crate -Des.cluster.name=cluster
+# docker run -d -p 4200:4200 -p 4300:4300 crate crate
 ```
 
-### Environment
+## Attach Persistent Data Directory
+
+Crate stores all important data in */data*. It's advised to mount this directory to avoid writing within the docker image:
 
-To set environment variables for Crate Data you need to use the `--env` option when starting the docker image.
+```console
+# docker run -d -v <data-dir>:/data crate crate
+```
+
+## Use Custom Crate Configuration
 
-For example, setting the heap size:
+Crate is controlled by a single configuration file which has sensible defaults already. If you derive your container from the Crate container, make sure to place your file inside it and let Crate know where to find it:
 
 ```console
-$ docker run -d -p 4200:4200 -p 4300:4300 --env CRATE_HEAP_SIZE=32g crate
+# docker run -d crate crate -Des.config=</path/to>/crate.yml
 ```
 
-## Multicast
+Other configuration settings may be specified upon startup using the `-D` option prefix. For example, configuring the cluster name by using system properties works like this:
 
-Crate uses multicast for node discovery by default. However, Docker does only support multicast on the same host. This means that nodes that are started on the same host will discover each other automatically, but nodes that are started on different hosts need unicast enabled.
+```console
+# docker run -d crate crate -Des.cluster.name=<my-cluster-name>
+```
+
+For further configuration options refer to the[Configuration](https://crate.io/docs/stable/configuration.html) section of our documentation.
 
-You can enable unicast in your custom `crate.yml`. See also: [Crate Multi Node Setup](https://crate.io/docs/en/latest/best_practice/multi_node_setup.html).
+## Environment
 
-Due to its architecture, Crate publishes the host it runs on for discovery within the cluster. Since the address of the host inside the docker container differs from the actual host the docker image is running on, you need to tell Crate to publish the address of the docker host for discovery.
+Crate recognizes environment variables like `CRATE_HEAP_SIZE` that need to be set with the `--env` option before the actual Crate core starts. You may want to[assign about half of your memory it](https://crate.io/docs/reference/en/latest/configuration.html#crate-heap-size) as a rule of thumb to Crate:
```

Comment:
Not sure I understand "You may want toassign about half of your memory it" -- is that sentence missing a word? ("memory _to_ it" perhaps?)